data_layer/da_layer.go

```diff
@@ -36,10 +38,11 @@ func NewDAManager(datype config.DAType, home string) *DAManager {
 		dalayer = &celestia.Celestia{
 			Root: home,
 		}
+	case config.Avail:
+		dalayer = avail.NewAvail(home)
```

Comment:
maybe break `newavail` to constructor and init config, like celestia?

Reply:
the init of celestia is for the light client. we don't have light client in avail